cpp/oneapi/dal/algo/knn/backend/model_impl.hpp

```diff
@@ -122,5 +129,66 @@ using brute_force_model_impl = detail::brute_force_model_impl<Task>;
 template <typename Task>
 using kd_tree_model_impl = detail::kd_tree_model_impl<Task>;
 
+namespace interop = dal::backend::interop;
```

Comment:
Suggest to move it to the function scope

Reply:
applied